server/src/routes/wishlist.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const WishLists = require('../db/mongo/models/wishlist');
+
+const router = express.Router();
+
+router.get('/:username', async (req, res) => {
+    const username  = req.params.username;
+    try {
+        let wishlist = await WishLists.find({ username: username}).exec();
```

Comment:
You might want to show the series actual values (no need for seasons/episodes) and not `series_ids` only - think about what the Client will want to get and display from this route.